e2e/testcases/cli_test.go

```diff
@@ -833,3 +848,235 @@ func TestNomosVetNamespaceRepo(t *testing.T) {
 		})
 	}
 }
+
+func TestCLIBugreportNomosRunningCorrectly(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.NomosCLI, ntopts.RequireGKE(t))
+	tmpDir := nomostest.TestDir(t)
+	tw := nomostesting.New(t, nomostesting.NomosCLI)
+	var bugReportZipName, bugReportDirName string
+	_ = nomostest.NewOptStruct(nomostest.TestClusterName(tw), tmpDir, tw)
+	t.Cleanup(func() {
+		// remove zip and folder
+		_ = os.Remove(bugReportZipName)
+		_ = os.RemoveAll(bugReportDirName)
+	})
+
+	// Confirm all the namespaces and pods are up
+	time, e := nomostest.Retry(nt.DefaultReconcileTimeout, func() error {
+		err := nt.Validate(KubeSystem, "", &corev1.Namespace{})
+		if err != nil {
+			tw.Fatal(err)
+		}
+		err = nt.Validate(ConfigManagementSystem, "", &corev1.Namespace{})
+		if err != nil {
+			tw.Fatal(err)
+		}
```

Comment:
Do sth to avoid the duplication of the following statements:
```
       err = nt.Validate(..., "", &corev1.Namespace{})
		if err != nil {
			tw.Fatal(err)
		}
```

Reply:
Using status.MultiError instead to reduce the error catch.